package.json

```diff
@@ -14,11 +14,18 @@
   },
   "devDependencies": {
     "eslint-config-custom": "*",
-    "prettier": "latest",
+    "prettier": "2.8.3",
     "turbo": "latest"
   },
   "engines": {
     "node": ">=14.0.0"
   },
+  "dependencies": {
+    "libphonenumber-js": "^1.10.18",
+    "node-fetch": "^2.6.1",
+    "passport-jwt": "^4.0.1",
+    "sass": "^1.57.1",
+    "validator": "^13.7.0"
+  },
```

Comment:
All these dependencies should be installed in specific directories, depending on which services use them (e.g. `apps/api`, ...).